libcloud/backup/providers.py

```diff
@@ -25,7 +25,9 @@
     Provider.GCE:
     ('libcloud.backup.drivers.gce', 'GCEBackupDriver'),
     Provider.DIMENSIONDATA:
-    ('libcloud.backup.drivers.dimensiondata', 'DimensionDataBackupDriver')
+    ('libcloud.backup.drivers.dimensiondata', 'DimensionDataBackupDriver'),
+    Provider.NTTCIS:
+    ('libcloud.backup.drivers.nttcis', 'NttCisNodeDriver'),
```

Comment:
This should surely be the backup driver, not the Node driver?

Reply:
Removed the NTTCIS backup driver from backup/providers and bacukp/types as it has been deprectaed.